bears/c_languages/AStyleLintBear.py

```diff
@@ -0,0 +1,56 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='astyle',
+        output_format='corrected',
+        use_stdin=True)
+class AStyleLintBear:
+    """
+    Artistic Style is a source code indenter, formatter,
+    and beautifier for the C, C++, C++/CLI, Objective‑C,
+    C# and Java programming languages.
+
+    For more information, consult <http://astyle.sourceforge.net/astyle.html>.
+    """
+
+    LANGUAGES = {'C', 'C++', 'Objective-C', 'C#', 'Java'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='astyle')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         astyle_bracket_style: str='',
```

Comment:
I am very certain of this because when I was implementing this long time ago, I set the default to allman and my tests that used config files kept failing. Then I realized that this happened because the config file options were been ignored since they were specified from the command line.